formula-android/src/main/java/com/instacart/formula/integration/ActivityHolder.kt

```diff
@@ -45,9 +45,7 @@ class ActivityHolder<Activity : FragmentActivity> {
     }
 
     fun detachActivity(activity: Activity) {
-        if (this.activity == activity) {
-            this.activity = null
-        }
+        this.activity = null
```

Comment:
This check is not needed since `StoreManager` already confirms they're the same

Reply:
I think I had this check in case the new activity is created before old one is destroyed.

Reply:
Ah interesting. I'll put back just in case.